src/templates/blog-category-list.js

```diff
@@ -10,52 +10,56 @@ import Footer from "../sections/General/Footer";
 
 import { GlobalStyle } from "../sections/app.style";
 import theme from "../theme/app/themeStyles";
```

Comment:
@kayceeDev is there any change other than formatting in this file?

Reply:
Yes. I made changed to the query of `blog.js` file. `Blog-tags-list.js` and `blog-category-list.js.` Then the search feature was then used in just the `blog-grid.js` and the `blog-list.js`